modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -139,16 +139,26 @@ For more information about how to resolve this error, see ``Registering Older Cl
 When you update {centos} clients, the packages do not include metadata about the updates.
 You can use a third-party errata service to obtain this information.
 
+[WARNING]
+====
+
 ifeval::[{suma-content} == true]
 
-[IMPORTANT]
-====
-The third-party errata service described here is provided and maintained by the community.
+The third-party errata service described here, CEFS, is provided and maintained by the community.
 It is not supported by {suse}.
-====
 
 endif::[]
 
+Please note that CEFS authors provide errata (patch) data on a best-effort basis, in the hope they will be useful but with no guarantees of correctness or currency.
+
+In particular, it is acknowledged that patch (errata) dates might be incorrect (https://github.com/stevemeier/cefs/issues/28#issuecomment-656579382) and it was verified in at least one case that published data was more than one month old (https://github.com/stevemeier/cefs/issues/28#issuecomment-656573607).
+
+Additionally, any defects in the service might result in unreliable patch information to be imported in your server. Consequently any reports, audits or other patch-related information produced in terms of currency of your systems, including CVE patch applicability, might be incorrect as a result.
+
+Please consider alternatives such as not relying on patch data or using different OS distributions depending on your security-related requirements and certifications criteria.
```

Comment:
```suggestion
The authors of CEFS provide patches and updates on a best-effort basis, in the hope they will be useful but with no guarantees of correctness or currency.
This could mean that the patch dates could be incorrect, and in at least one case, the published data was shown to be more than a month old.
For more information on these cases, see https://github.com/stevemeier/cefs/issues/28#issuecomment-656579382 and https://github.com/stevemeier/cefs/issues/28#issuecomment-656573607.

Any problems or delays with the patch data might result in unreliable patch information being imported to your {productname} Server. 
This would cause reports, audits, CVE updates, or other patch-related information to also be incorrect.
Please consider alternatives to using this service, such as independently verifying patch data, or choosing a different operating system, depending on your security-related requirements and certifications criteria.
```